romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -157,7 +168,7 @@ def process(self, input):
             buffsize = ramp_fit.BUFSIZE
             image_info, integ_info, opt_info, gls_opt_model = ramp_fit.ramp_fit(
                 input_model, buffsize, self.save_opt,
-                readnoise_model.data, gain_model.data, self.algorithm,
+                readnoise_model.data.value, gain_model.data.value, self.algorithm,
```

Comment:
These would be better with .to()'s, in my opinion.

Reply:
Relegating to #644.